src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -38,42 +42,72 @@ For detailed information about these settings, how to make them permanent, and i
 
 ## Hardware considerations
 
-### Network
+As load increases, you need to pay attention to hardware constraints.
 
-Network throughput of the machine is an important consideration when running large tests.
-Many AWS EC2 machines come with 1Gbit/s connection which may limit the amount of load k6 can generate.
+<DescriptionList>
 
-When running the test, you can use `iftop` in the terminal to view in real-time the amount of network traffic generated.
+Network
+: Network throughput of the machine is an important consideration when running large tests.
+Many AWS EC2 machines come connections of with 1Gbit/s, which may limit the load k6 can generate.
+: When running the test, use a tool like `iftop` in the terminal to view the amount of network traffic generated in real time.
 If the traffic is constant at 1Gbit/s, your test is probably limited by the network card. Consider upgrading to a different EC2 instance.
 
-### CPU
-
-Unlike many other load testing tools, k6 is heavily multi-threaded. It will effectively use all available CPU cores.
-
-The amount of CPU you need depends on your test files (sometimes called test script).
+CPU
+: Unlike many other load testing tools, k6 is heavily multi-threaded. It will effectively use all available CPU cores.
+: The amount of CPU you need depends on your test script and associated files.
 Regardless of the test file, you can assume that large tests require a significant amount of CPU power.
```

Comment:
It's a feature of extended markdown: https://www.markdownguide.org/extended-syntax#definition-lists
I don't like the syntax, but it's gained adoption.


The parsers of many SSGs use it, so it'd work in Hugo out of the box. Our Gatsby component has the disadvantage of requiring a `<DescriptionList>` tag, but the advantage of automatically generating `id` attributes, so each term is has an anchor.

I don't think it works it Github though
: does it?

Reply:
Ah, got it. I just noticed it's not just line breaks, but separate elements.

Then I'd argue we're abusing the purpose of _definition_ lists, just because of how they're currently styled. It's akin to using tables for layout in the old days, and goes against the semantic HTML concept.

There's no reason these couldn't be wrapped in an element just like `<DescriptionList>`, but contain plain `h3`s or whatever level is appropriate, and then the paragraphs could just have a small `padding-left`. It would require trivial amounts of CSS, the markup would be much simpler, and you could just do a double line break as usual to get new paragaphs. And we wouldn't need to fix issues like #1002, at least not right now.

I don't want to block this PR from merging because of this, but I can create an issue if you agree.

Reply:
I don't think it's abusing a `<dl`>`. It is actually called a _description list_ by MDN. I swear it used to be called definition list and then they broadened the name. I'm not going to dig, but the standard says the lists can be any name-value groups:

```
Name-value groups may be terms and definitions, metadata topics and values, questions and answers, or any other groups of name-value data.
```

In these cases, I think the description list works well semantically, because these all are lists in one distinct category, e.g. "Specific hardware considerations".  In another form, the info could be represented as a table, since their structure is so similar. On the other hand, the grouping of h3s can be a little looser and usually don't have the same parallellism between heading and content.

But, this is very, very pedantic and I'm being a little bit vague and mystical. Still, I don't think it's so important and I don't think we're doing anything that abuses web conventions or harms accessibility. If you think we are, please make an issue.

Reply:
Fair enough, the naming is confusing, but all the usage examples point to this being used for defining or describing terms, for metadata, and generally simpler usage than what we're using them for (#1002 would be something we shouldn't need).

The reason the colon is used is precisely to define the term, and that's not our use case here. On the glossary page it makes perfect sense, but rarely elsewhere, and definitely not just for indentation.

I'd also point out [this note from MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#notes):

> Do not use this element (nor `<ul>` elements) to merely create indentation on a page. Although it works, this is a bad practice and obscures the meaning of description lists.
>
> To change the indentation of a description term, use the CSS `margin` property.

Which is exactly what we're doing here. It also has accessibility implications for screen readers, etc., and we don't need to complicate things.

So let's merge this as is, but I'll create the issue to change this.